nbformat/v4/nbformat.v4.schema.json

```diff
@@ -373,7 +373,7 @@
                   "$ref": "#/definitions/misc/multiline_string"
                 },
                 "patternProperties": {
-                    "^application/([a-zA-Z0-9.]+\\+)?json$": {
+                    "^application/([a-zA-Z0-9.-]+\\+)?json$": {
```

Comment:
It's working for me locally. The `.` is not escaped so I don't think these types of characters need to be escaped within square brackets.